packages/image/src/image.tsx

```diff
@@ -0,0 +1,62 @@
+import { forwardRef, type ElementRef, type ComponentProps } from "react";
+import { getImageAttributes, type ImageLoader } from "./image-optimize";
+
+const defaultTag = "img";
+
+type ImageProps = ComponentProps<typeof defaultTag> & {
+  quality?: number;
+  optimize?: boolean;
+  loader: ImageLoader;
+};
+
+export const Image = forwardRef<ElementRef<typeof defaultTag>, ImageProps>(
+  ({ quality, loader, optimize, ...imageProps }, ref) => {
+    // Temporary set to false, to support previous image behaviour
+    const DEFAULT_OPTIMIZE = false;
+
+    const imageAttributes = getImageAttributes({
+      src: imageProps.src,
+      srcSet: imageProps.srcSet,
+      sizes: imageProps.sizes,
+      width: imageProps.width,
+      quality,
+      loader,
+      optimize: optimize ?? DEFAULT_OPTIMIZE,
+    }) ?? { src: imagePlaceholderSvg };
+
+    return (
+      <img {...imageProps} {...imageAttributes} decoding="async" ref={ref} />
+    );
+  }
+);
+
+Image.defaultProps = {
+  src: "",
+  loading: "lazy",
+};
```

Comment:
defaultProps is legacy and may be eventually be removed in favour of destructuring defaults

Reply:
We know there is another PR exists, will fix all defaultProps there